### PR TITLE
Implement bookmark flush API

### PR DESCRIPTION
**Pull request by fbiville**

### CODE REVIEW

File: neo4j/bookmarks_test.go [hunk 1]
```diff
@@ -197,6 +197,25 @@ func TestBookmarkManager(outer *testing.T) {
...
+
+		bookmarkManager.Forget("db", "par")
+
+		allBookmarks := bookmarkManager.GetAllBookmarks()
+		AssertEqualsInAnyOrder(t, allBookmarks, []string{"bar", "fighters"})
+		AssertIntEqual(t, len(bookmarkManager.GetBookmarks("db")), 0)
+		AssertEqualsInAnyOrder(t, bookmarkManager.GetBookmarks("foo"),
+			[]string{"bar", "fighters"})
+		AssertIntEqual(t, len(bookmarkManager.GetBookmarks("par")), 0)
+	})
```
Comment (bigmontz) [inline neo4j/bookmarks_test.go]: Maybe check if it doesn't break if you try to forget something you don't remember.

**Review by bigmontz — APPROVED** [review]
🤯

**Review by robsdedude — APPROVED** [review]
:brain: :wastebasket:

File: neo4j/bookmarks_test.go (hunk 1 above)
Comment (fbiville) [reply neo4j/bookmarks_test.go]: I need to print a t-shirt with this sentence :D

Comment (fbiville) [reply neo4j/bookmarks_test.go]: fixed in 08543b390283f442e50a55178c13e43adbe94650

**Pull request merged**